app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -107,14 +107,14 @@ class MainActivity : AppCompatActivity() {
     This button trigger a basics notification in 1 sec
     here we use an id based on current time. We may use some parsed part of the corresponding deadline later.
     */
-
     fun triggerNotification(view: View) {
         DeadlineNotification().setNotification(
-            Deadline(
-                "notifDeadline",
-                DeadlineState.TODO,
-                LocalDateTime.now()
-            ), this, System.currentTimeMillis() + 1000
+            Pair(
```

Comment:
Your changes are fine but while you're at it, could you reformat this file? (`import java.time.LocalDate` is not used anymore).